src/common/tracer.cc

```diff
@@ -10,8 +10,8 @@
 
 namespace tracing {
 
-const opentelemetry::nostd::shared_ptr<opentelemetry::trace::Tracer> Tracer::noop_tracer = opentelemetry::trace::Provider::GetTracerProvider()->GetTracer("no-op", OPENTELEMETRY_SDK_VERSION);
-
+const opentelemetry::nostd::shared_ptr<opentelemetry::trace::NoopTracer> Tracer::noop_tracer = opentelemetry::nostd::shared_ptr<opentelemetry::trace::NoopTracer>(new opentelemetry::trace::NoopTracer);
```

Comment:
why did you change the creation of the noop_tracer?

Reply:
I wanted to drop those calls to opentelemetry SDK, but you are right. it happens only once